src/rdo.rs

```diff
@@ -596,11 +596,18 @@ impl DistortionScale {
 
   /// Binary logarithm in Q11
   #[inline]
-  pub const fn blog32(self) -> i16 {
+  pub const fn blog16(self) -> i16 {
```

Comment:
was this named incorrectly before?

Reply:
ah nevermind I see how this works

Reply:
Yes, I neglected to rename it when changing the output precision and word size.